appcues/src/main/java/com/appcues/data/model/ExperiencePrimitive.kt

```diff
@@ -93,6 +93,16 @@ internal sealed class ExperiencePrimitive(
         override val textDescription: String = items.joinTextDescriptions()
     }
 
+    data class CustomComponentPrimitive(
+        override val id: UUID,
+        override val style: ComponentStyle,
+        val identifier: String,
+        val config: AppcuesConfigMap,
+    ) : ExperiencePrimitive(id, style) {
+
+        override val textDescription: String? = null
```

Comment:
is this used for step interaction analytics on tap? maybe could use the `identifier` here to give something useful

Reply:
not really because the default tap is disabled for this type of object, so this textDescription is not doing anything really.